src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -549,7 +549,7 @@ public AnnotationViewModel(WorkspaceViewModel workspaceViewModel, AnnotationMode
         internal void SetGroupInputPorts()
         {
             InPorts.Clear();
-            List<ProxyPortViewModel> newPortViewModels;
+            List<PortViewModel> newPortViewModels;
```

Comment:
Can you sum this up?

Reply:
What are you asking here @mjkkirschner

Reply:
sorry @saintentropy - why did you make all these changes related to proxy type.

Reply:
I think instead of additional type, @saintentropy is using a property on `PortViewModel` to indicate if it is a proxy.. Given there is no new property in `ProxyPortViewModel` yet, this should be OK. But in the future if we need to store additional info, this may not scale. But I am fine with it for now, How about you @mjkkirschner?

Reply:
@mjkkirschner @QilongTang I maid the change as the purpose of the new wrapper class ProxyPortViewModel was just to provide a type check and did not have a different feature set.  Because there are now essentially two base classes for ports it seemed to make more sense to add the proxy status as a property (for the is proxy check) and keep the classes a bit simpler in the code.